clarity/contracts/arkadiko-auction-engine-v4-1.clar

```diff
@@ -378,7 +378,7 @@
     (auction (get-auction-by-id auction-id))
     (debt-left (- (get debt-to-raise auction) (get total-debt-burned auction)))
     (collateral-left (- (get collateral-amount auction) (get total-collateral-sold auction)))
-    (collateral-price (unwrap-panic (get-collateral-discounted-price oracle auction-id)))
+    (collateral-price (unwrap! (get-collateral-discounted-price oracle auction-id)))
```

Comment:
@nieldeckx you need to add a specific error here at the end

Reply:
Oops, done. :)